config/checkstyle_checks.xml

```diff
@@ -122,6 +122,7 @@
 
   <!-- Size Violations -->
   <module name="FileLength">
+    <property name="max" value="2500"/>
```

Comment:
The tests I added resulted in increase of code lines greater than 2000 in the IndentationCheckTest file.
Should I instead use suppression?